pages/reference/base-images/base-images.md

```diff
@@ -19,18 +19,19 @@ excerpt: Docker images maintained by balena
 - Multiple Distributions:
   - [Debian](https://www.debian.org/): buster (10), bullseye (11), bookworm (12) and sid
   - [Alpine](https://alpinelinux.org/): 3.12, 3.13, 3.14, 3.15, 3.16, 3.17 and edge
```

Comment:
```suggestion
  - [Alpine](https://alpinelinux.org/): 3.12, 3.13, 3.14, 3.15, 3.16, 3.17 and edge
```

If you can, can you please update the other entries as well! https://endoflife.date/alpine

Reply:
Ok sure no problem